### ISSUE TITLE
Fonts special characters don't show when start with ubuntu shortcut

### ISSUE THREAD

**Issue by Seybo** [jwilm/alacritty#1263]
> Which operating system does the issue occur on?

Ubuntu 16.04

> If on linux, are you using X11 or Wayland?

I'm not sure but 
```
_  ~ dpkg -l x11
dpkg-query: no packages found matching x11
_  ~ dpkg -l Wayland
dpkg-query: no packages found matching Wayland
```
If i start Alacritty normally (through the Applications -> Alacritty) and then run `tmux` special font characters show ok
![image](https://user-images.githubusercontent.com/10584167/39106060-78fbdcf4-46e3-11e8-9f5d-f2435f672a2f.png)
If I start it from the default ubuntu teminal with tmux `alacritty -e tmux` they also show.
But if i use my ubuntu shortcut with the same command for it, they dont:
![image](https://user-images.githubusercontent.com/10584167/39106131-e189aca6-46e3-11e8-879d-3678cfa3731b.png)
![image](https://user-images.githubusercontent.com/10584167/39106141-ed84c338-46e3-11e8-9280-3939070a57f8.png)

**Comment by chrisduerr** [comment]
When running `locale` in both the broken and the working terminal, do you get the same results?

**Comment by Seybo** [comment]
yep

**Comment by jwilm** [comment]
In the case where you start alacritty from the shortcut without `-e tmux` and later start `tmux`, your bash profile (or whatever shell profile) is evaluated. My guess is that this changes the environment tmux is launched in which is somehow responsible for your fonts working vs not.

As another quick sanity check, does your shortcut work as expected without `-e tmux`?

**Comment by Seybo** [comment]
yeah it does work without `-e tmux`. Ok, thanks guys but i thing we can close it anyway. I decided to try alacritty because i thought it might help me to reduce lagging on opening a large files. But couldn't get any different results compared to the default gnome terminal
